voip_oca/models/voip_oca_call.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class VoipOcaCall(models.Model):
+    _name = "voip.oca.call"
```

Comment:
I don't think we need `oca` on the model (and if Odoo is calling this the same, it's not a problem, as both modules are incompatible between them and we shouldn't look for the compatibility IMO).

```suggestion
    _name = "voip.call"
```